cypress/fixtures/docs/staticJSONObjects.md

```diff
@@ -0,0 +1,144 @@
+# Static JSON Objects
+
+Firebolt Calls has two types of JSON objects: Firebolt Calls and Validation Objects. These objects are used to make API calls and validate the response received from the API.
+
+## [Supported Static Glue Codes](../../support/step_definitions/fireboltCalls.md)
+
+## Firebolt Calls
+
+Firebolt object is used to make an API call. It contains the following parameters:
+
+- method - Name of the API to make an API call
+- params - Represent the parameters to be sent for a Firebolt call
+- context - Represent the data that needs to be stored in the API/event object that helps to search for a specific object which contains the response in a global list
+- expected - Determine whether expecting for an error or result
+
+### How to Configure Firebolt Calls
+
+- Define a firebolt call object in `cypress/fixtures/fireboltCalls/<fileName>.json`, where fileName can be any name.
+- Add the method, params, context, and expected parameters to the firebolt call object. params, context, and expected are optional parameters. By default, they are set to below values:
+  - params: {}
+  - context: {}
+  - expected: "result"
+- `params` is the data that needs to be sent in the API call. It can be a string, number, boolean, or an object. Params can be a string or object
+  - If the params passed as a objects, it will be used as is in the API call.
+  - If the params passed as a string, and these values are processed as below:
+    - If the parameters is boolean, number, or any other type, and these values are used across features, then add them in the [defaultTestDataSet.json](../defaultTestData.json) file and use the key name in the params.
+    - If want to pass complex parameters. Define the params inside the `cypress/fixtures/modules/<fileName>.json` file and use the key name in the params field.
+
```

Comment:
can you give a one line here about each example

Reply:
addressed the comments